planck-cljs/src/planck/io.cljs

```diff
@@ -346,12 +346,67 @@
 (defn ^boolean directory?
   "Checks if dir is a directory."
   [dir]
-  (js/PLANCK_IS_DIRECTORY (:path (as-file dir))))
+  (boolean (js/PLANCK_IS_DIRECTORY (:path (as-file dir)))))
 
 (s/fdef directory?
   :args (s/cat :dir (s/or :string string? :file file?))
   :ret boolean?)
 
+(defn exists?
+  "Checks if f exists on disk."
+  [f]
```

Comment:
I wonder if there is a better argument name here: `f`is often used for functions. Perhaps `file`? Hrm.

Reply:
Looks like `file-attributes` names its arguments `path`, FWIW. Not sure if I like that one either, but `path` seems better than `f`.

Reply:
Makes sense to me, though I think that would imply a larger change throughout `planck.io`.  Would you like me to make that change, or is that better handled separately, after this PR is merged?

Reply:
No need to change the argument names now. Not that big of a deal.

Reply:
Issue #870.